scripting/get5/eventlogger.sp

```diff
@@ -127,18 +127,18 @@ public void EventLogger_GoingLive() {
 }
 
 public void EventLogger_PlayerDeath(int killer, int victim, bool headshot, int assister,
-                             int flash_assister, const char[] weapon) {
+                             bool flash_assist, const char[] weapon) {
   EventLogger_StartEvent();
   AddMapData(params);
   AddPlayer(params, "attacker", killer);
   AddPlayer(params, "victim", victim);
   params.SetInt("headshot", headshot);
   params.SetString("weapon", weapon);
 
-  if (assister > 0)
+  if (assister > 0) {
     AddPlayer(params, "assister", assister);
-  if (flash_assister > 0)
-    AddPlayer(params, "flash_assister", flash_assister);
+    params.SetBool("flash_assist", flash_assist);
+  }
 
```

Comment:
I know this is a breaking change, but I don't know how much semantic versioning you normally apply to the event-logger.